packages/aws-cdk-lib/aws-lambda-nodejs/lib/function.ts

```diff
@@ -90,6 +94,17 @@ export interface NodejsFunctionProps extends lambda.FunctionOptions {
    * @default - the directory containing the `depsLockFilePath`
    */
   readonly projectRoot?: string;
+
+  /**
+   * The code that will be deployed to the Lambda Handler. If included, then properties related to
+   * bundling of the code are ignored. In the constructor of NodeJsFunction, where the Super is called,
+   * you can see which bundling properties are ignored.
```

Comment:
Let's re-phrase this. Maybe just leave it to "If included, then properties related to bundling of the code are ignored". The "In the constructor of the NodeJsFunction, where the Super is called you can see which bundling properties are ignored" isn't helpful since I wouldn't necessarily expect a user to have to go into our source code.

Reply:
that makes sense, agreed